ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/ImageCvePageHeader.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { gql } from '@apollo/client';
+import { Button, Flex, LabelGroup, Label, Skeleton, Text, Title } from '@patternfly/react-core';
+import { getDateTime } from 'utils/dateUtils';
+
+export type ImageCveMetadata = {
+    cve: string;
+    firstDiscoveredInSystem: Date | null;
+};
+
+export const imageCveMetadataFragment = gql`
+    fragment ImageCVEMetadata on ImageCVECore {
+        cve
+        # TODO summary
+        # TODO url
+        firstDiscoveredInSystem
+    }
+`;
+
+export type ImageCvePageHeaderProps = {
+    data?: ImageCveMetadata;
+};
+
+function ImageCvePageHeader({ data }: ImageCvePageHeaderProps) {
+    return data ? (
+        <Flex direction={{ default: 'column' }} alignItems={{ default: 'alignItemsFlexStart' }}>
+            <Title headingLevel="h1" className="pf-u-mb-sm">
+                {data.cve}
+            </Title>
+            <LabelGroup numLabels={1}>
+                <Label isCompact>
+                    First discovered in system {getDateTime(data.firstDiscoveredInSystem)}
+                </Label>
+            </LabelGroup>
+            <Text>
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam vel aliquet velit.
```

Comment:
I tried getting chat-gpt to generate "software vulnerability style lorem ipsum", but the result was disappointing.